app_streaming.py

```diff
@@ -0,0 +1,71 @@
+from fastapi import FastAPI
+from llama_cpp import Llama
+from transformers import pipeline
+from pydantic import BaseModel
+from fastapi.responses import StreamingResponse
+from typing import Optional
+
+# import uvicorn
+
+
+class Message(BaseModel):
```

Comment:
I think the name `Message` is a bit misleading and can be mistaken with the `message` object in the `messages` list in the `/chat/completion` api. Maybe something like `GenerationRequest` is more suiting